src/App.css

```diff
@@ -111,7 +111,7 @@ body {
   border: none;
   background-color: #7565ce;
   color: #fff;
-  font-family: 'Inter', sans-serif;
+  font-family: 'Anuphan', sans-serif;
```

Comment:
- [ ] Kindly implement the styling to the nav items likewise, when the user is on that specific screen and when not is what the greyness specifies.

- Your design
<img width="192" alt="image" src="https://user-images.githubusercontent.com/41163571/233631473-b6ff4f94-574f-4915-83ed-69871edde6b9.png">

- Zeplin design
<img width="125" alt="image" src="https://user-images.githubusercontent.com/41163571/233631384-da9658af-338d-45a4-b27f-575de00a53f7.png">